scania-digital-design-system/sdds
# fix(divider): move theme rules to component folder

**Describe pull-request** 

Moving theming rules to component folder 

**Solving issue** 

Fixes: [AB#2216](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2216) 

**How to test** 

1. Open divider in Storybook of this branch 
2. Open divider in Storybook of production 
3. Compare styling- there should not be any difference between those two
